terracumber_config/tf_files/SUSEManager-5.0-build-validation-PRV.tf

```diff
@@ -0,0 +1,1561 @@
+  images      = [ "almalinux9o", "centos7o", "oraclelinux9o", "rocky8o", "rocky9o" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br1"
+  }
+}
+
+module "base_new_sle" {
+  providers = {
+    libvirt = libvirt.ginfizz
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-bv-50"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles15sp1o", "sles15sp2o", "sles15sp3o", "sles15sp4o", "sles15sp5o", "slemicro51-ign", "slemicro52-ign", "slemicro53-ign", "slemicro54-ign", "slemicro55-ign" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br1"
+  }
+}
+
+module "base_retail" {
+  providers = {
+    libvirt = libvirt.hugo
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-bv-50"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles12sp5o", "sles15sp3o", "sles15sp4o", "opensuse154o" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br1"
+    additional_network = "192.168.50.0/24"
+  }
+}
+
+module "base_debian" {
+  providers = {
+    libvirt = libvirt.irishcoffee
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-bv-50"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "ubuntu2204o", "debian10o", "debian11o", "debian12o" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br1"
+  }
+}
+
+// WORKAROUND: overdrive4 will be replaced with a new ARM server
+//module "base_arm" {
+//  providers = {
+//    libvirt = libvirt.overdrive4
+//  }
+//
+//  source = "./modules/base"
+//
+//  cc_username = var.SCC_USER
+//  cc_password = var.SCC_PASSWORD
+//  name_prefix = "suma-bv-50"
+//  use_avahi   = false
+//  domain      = "mgr.prv.suse.net"
+//  images      = [ "opensuse154armo", "opensuse155armo" ]
+//
+//  mirror = "minima-mirror-ci-bv.mgr.prv.suse.net"
+//  use_mirror_images = true
+//
+//  testsuite = true
+//
+//  provider_settings = {
+//    pool        = "ssd"
+//    bridge      = "br1"
+//  }
+//}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "srv"
+  image              = "slemicro55-ign"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:a1"
```

Comment:
In infra, the server mac address is aa:b2:92:42:00:01

Reply:
Nice catch. Thank you, Maxime.

Reply:
Fixed.